src/Settings/Builders/Sphinx.hs

```diff
@@ -16,7 +16,7 @@ sphinxBuilderArgs = do
                 , arg =<< getInput
                 , arg outPath ]
             , builder (Sphinx Man) ? mconcat
-                [ arg "-b", arg "latex"
+                [ arg "-b", arg "man"
```

Comment:
This does look like the right change though.

@patrickdoc Was this a typo?

Reply:
Ah, yes, my apologies. That should indeed be `man` not `latex`.